src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatTile.tsx

```diff
@@ -24,36 +23,39 @@ export interface KeyStatConfig {
 const KeyStatTile = ({
   dataBlockRequest,
   summary,
-  dataBlockResponse: response,
   children,
   handleApiErrors,
 }: KeyStatProps) => {
   const [dataBlockResponse, setDataBlockResponse] = useState<
     DataBlockResponse | undefined
-  >(response);
-
-  const [config, setConfig] = useState<KeyStatConfig | undefined>();
+  >();
 
   useEffect(() => {
-    if (!dataBlockResponse) {
-      DataBlockService.getDataBlockForSubject({
-        ...dataBlockRequest,
-        includeGeoJson: false,
-      })
-        .then(setDataBlockResponse)
-        .catch(handleApiErrors);
-    } else {
+    DataBlockService.getDataBlockForSubject({
+      ...dataBlockRequest,
+      includeGeoJson: false,
+    })
+      .then(setDataBlockResponse)
+      .catch(handleApiErrors);
```

Comment:
You don't need to explicitly handle API errors anymore - it is done by default after one of Nick's PRs last week

Reply:
I'm now getting handleApiErrors with useErrorControl -- is that right?